includes/admin/class-wc-rest-upe-flag-toggle-controller.php

```diff
@@ -0,0 +1,12 @@
+<?php
+/**
+ * Class WC_REST_UPE_Flag_Toggle_Controller
+ *
+ * @package WooCommerce\Payments\Admin
+ */
+
+/**
+ * REST controller for UPE feature flag. Needs to stay in the codebase to avoid error on plugin update for versions 6.9.2 or earlier.
+ */
+class WC_REST_UPE_Flag_Toggle_Controller extends WP_REST_Controller {
+}
```

Comment:
While this fixes the error, it still throws a notice:

```
Notice: Function WP_REST_Controller::register_routes was called incorrectly. Method 'WP_REST_Controller::register_routes' must be overridden. Please see Debugging in WordPress for more information. (This message was added in version 4.7.0.) in /srv/users/user4dec5d51/apps/user4dec5d51/public/wp-includes/functions.php on line 6031
```

Can be fixed with this:

```suggestion
class WC_REST_UPE_Flag_Toggle_Controller extends WP_REST_Controller {
    public function register_routes() {}
}
```

Reply:
Oh, for some reason, I haven't seen this error. Thank you, applied the fix now!

Reply:
Alternatively, I could stop extending the controller, but I prefer to avoid other potential issues coming from it so I applied the proposed changes.